src/activities/assignments/AssignmentEntity.js

```diff
@@ -258,4 +258,28 @@ export class AssignmentEntity extends Entity {
 		];
 		await performSirenAction(this._token, action, fields);
 	}
+
+	annotationToolsAvailable() {
```

Comment:
I'd recommend `getAvailableAnnotationTools` here - just to keep consistent the `get` nomenclature elsewhere

Reply:
Is `_getAvailableAnnotationTools` preferred over `getAvailableAnnotationTools`?

Reply:
Depends - is this being used by your consumer as well? (The underscore is just being used to denote private methods by convention, really.)

Reply:
Yes it is to set the checkbox state, so `getAvailableAnnotationTools`?

Reply:
Sounds right to me!